ledger/src/blockstore_processor.rs

```diff
@@ -1176,16 +1192,25 @@ pub struct ThreadExecuteTimings {
 }
 
 impl ThreadExecuteTimings {
-    pub fn report_stats(&self, slot: Slot) {
+    pub fn report_stats(&self, slot: Slot, is_unified_scheduler_enabled: bool) {
```

Comment:
So I'm a bit confused by the naming/intent of the argument here w/ the use.

In the calling code we're calling it `are_stats_from_completed_scheduler` which to me indicates that:
1. we're using unified scheduler
2. the scheduler is completed

But the argument of this function `is_unified_scheduler_enabled` is only the first condition "we're using unified scheduler".

I'm like 90% sure that if using unified scheduler, at this point when we `wait_for_completed_scheduler` it will return a `Some`, but it seems possible it does not, at which point our metrics are misreported because `is_unified_scheduler_enabled` is `false`.

Similar concern about intent on `accumulate` fn.